editor/plugins/org.fusesource.ide.wsdl2rest.ui/src/org/fusesource/ide/wsdl2rest/ui/wizard/Wsdl2RestWizard.java

```diff
@@ -0,0 +1,359 @@
+		one.setVersion("1.0.0.Final-redhat-1"); //$NON-NLS-1$
+		deps.add(one);
+		IProject project = options.getProject();
+		new MavenUtils().updateMavenDependencies(deps, project);
+	}
+
+	private void prepare(IFolder folder) throws CoreException {
+		if (!folder.exists() && folder.getParent() instanceof IFolder) {
+			prepare((IFolder) folder.getParent());
+			folder.create(false, true, null);
+		}
+	}
+	
+	private IResource getResourceForPath(IProject project, IPath path) throws CoreException {
+		IResource resource = ResourcesPlugin.getWorkspace().getRoot().findMember(path);
+		if (resource == null) {
+			if (!path.isEmpty() && path.segment(0).equals(options.getProjectName())) {
+				path = path.removeFirstSegments(1);
+			}
+			IFolder folder = project.getFolder(path);
+			if (!folder.exists()) {
+				prepare(folder);
+			}
+			resource = folder;
+		}
+		return resource;
+	}
+	
+	private IResource getFileForPath(IProject project, IPath path) throws CoreException {
+		if (path.getFileExtension() == null) {
+			return getResourceForPath(project, path);
+		}
+		IResource resource = ResourcesPlugin.getWorkspace().getRoot().findMember(path);
+		if (resource == null) {
+			if (!path.isEmpty() && !path.segment(0).equals(options.getProjectName())) {
+				path = project.getFullPath().append(path);
+			}
+			resource = ResourcesPlugin.getWorkspace().getRoot().findMember(path);
+			if (resource == null) {
+				IPath absolute = ResourcesPlugin.getWorkspace().getRoot().getLocation().append(path).makeAbsolute();
+				java.io.File ioFile = absolute.removeLastSegments(1).makeAbsolute().toFile();
+				ioFile.mkdirs();
+				project.refreshLocal(IResource.DEPTH_INFINITE, new NullProgressMonitor());
+				IFile newFile = ResourcesPlugin.getWorkspace().getRoot().getFileForLocation(absolute);
+				new CamelIOHandler().loadCamelModel(newFile, new NullProgressMonitor());				
+				resource = newFile;
+			}
+		}
+		return resource;
+	}
+	
+	/**
+	 * Use the settings collected and call the wsdl2rest utility.
+	 *
+	 * @throws Exception
+	 */
+	private void generate() throws Exception {
+		URL wsdlLocation = new URL(options.getWsdlURL());
+		IPath javaPath = new org.eclipse.core.runtime.Path(options.getDestinationJava());
+		IProject project = options.getProject();
+		IResource resource = getResourceForPath(project, javaPath);
+		File javaFile = findFileInEFS(resource);
+
+		// use project to determine if we are building a spring or blueprint project
+		boolean isBlueprint = isProjectBlueprint(project);
+		boolean isSpringBoot = isProjectSpringBoot(project);
+		
+		IPath camelPath = new org.eclipse.core.runtime.Path(options.getDestinationCamel());
+		IResource camelResource = getFileForPath(project, camelPath);
+		File camelFile = findFileInEFS(camelResource);
+		
+		if (javaFile != null) {
+			ClassLoader oldLoader = Thread.currentThread().getContextClassLoader();
+			try {
+				// initialize bus using bundle classloader, to prevent project dependencies from leaking in
```

Comment:
I still don't understand this comment.
Which bus is initialized?
Why "project dependencies from leaking in"?

Reply:
Was copied from the Switchyard wrapper when I created this plugin. Can remove it.